arch/x86/core/intel64/cpu.c

```diff
@@ -46,6 +46,42 @@ struct x86_cpuboot x86_cpuboot[] = {
 	LISTIFY(CONFIG_MP_MAX_NUM_CPUS, X86_CPU_BOOT_INIT, (,)),
 };
 
+int x86_update_cpu_info(uint8_t cpu_id)
```

Comment:
I don't see why this patch is in this PR?  It doesn't have anything to do with CPU count.  Can you do this as a separate submission?